assets/src/edit-story/components/library/panes/text/textSets/index.js

```diff
@@ -14,12 +14,54 @@
  * limitations under the License.
  */
 
+/**
+ * External dependencies
+ */
+import { useEffect, useState } from 'react';
+import styled from 'styled-components';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
 /**
  * Internal dependencies
  */
-import getTextSets from './getTextSets';
+import { Section } from '../../../common';
+import { UnitsProvider } from '../../../../../units';
+import { PAGE_RATIO, TEXT_SET_SIZE } from '../../../../../constants';
+import { getTextSets } from './utils';
+import TextSet from './textSet';
+
+const TextSetContainer = styled.div`
```

Comment:
This should probably also be a list - and with the section title as `aria-labelledby`.

It doesn't have to be an `<ul>`, `role="list"` (with corresponding `role="listitem"` for the items) would work too.

Reply:
This as well is currently addressed a bit in #4478 instead, LMK if that's OK with you.